### PR TITLE
Minor doc improvements for #318

### PR DESCRIPTION
**Pull request by jonas**
First batch of changes to improve the Markdown.

### CODE REVIEW

File: akka-http/src/main/resources/reference.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#######################################
+###################################
 # akka-http Reference Config File #
-#######################################
+###################################
```
Comment (ktoso) [inline akka-http/src/main/resources/reference.conf]: yay! no more warnings I suppose?

File: docs/src/main/paradox/java/http/client-side/host-level.md [hunk 2]
```diff
@@ -53,9 +53,9 @@ the same pooled connections.
 
 The "pool client flow" returned by `Http.get(system).cachedHostConnectionPool(...)` has the following type:
 
-```
+```java
 // TODO Tuple2 will be changed to be `akka.japi.Pair`
-Flow[Tuple2[HttpRequest, T], Tuple2[Try[HttpResponse], T], HostConnectionPool]
+Flow<Tuple2<HttpRequest, T>, Tuple2<Try<HttpResponse>, T>, HostConnectionPool>
```
Comment (ktoso) [inline docs/src/main/paradox/java/http/client-side/host-level.md]: good catch

File: docs/src/main/paradox/scala/http/routing-dsl/testkit.md [hunk 3]
```diff
@@ -5,9 +5,11 @@ For services built with the Routing DSL Akka HTTP provides a dedicated testkit t
 route logic easy and convenient. This "route test DSL" is made available with the *akka-http-testkit* module.
 To use it include the following dependency:
 
+@@@vars
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/routing-dsl/testkit.md]: ok, so that's how it works now.

**Comment by ktoso** [comment]
OK TO TEST

File: docs/src/main/paradox/scala/http/routing-dsl/testkit.md (hunk 3 above)
Comment (jonas) [inline docs/src/main/paradox/scala/http/routing-dsl/testkit.md]: Yes, however it looks like we have to either patxh Paradox or rewrite the Spray JSON support section to work with this style.

File: docs/src/main/paradox/java/http/client-side/request-level.md [hunk 4]
```diff
@@ -58,5 +58,5 @@ lack of entity consumption as a back-pressure signal and stop reading from the u
 This is a feature of Akka HTTP that allows consuming entities (and pulling them through the network) in
 a streaming fashion, and only *on demand* when the client is ready to consume the bytes -
 it may be a bit surprising at first though.
-There are tickets open about automatically dropping entities if not consumed ([#18716](https://github.com/akka/akka/issues/18716) and [#18540](https://github.com/akka/akka/issues/18540)),
+There are tickets open about automatically dropping entities if not consumed ([#183](https://github.com/akka/akka-http/issues/183) and [#117](https://github.com/akka/akka-http/issues/117)),
```
Comment (ktoso) [inline docs/src/main/paradox/java/http/client-side/request-level.md]: uff, great catch!

**Comment by ktoso** [comment]
Great catches here, thanks a lot!
WDYT, when should we delete the rst files?
You can do that in order to get massive negative contrib credit (lines removed is the best street cred 😎 )

**Comment by ktoso** [comment]
Refs #318

**Pull request merged**